### PR TITLE
Dashboard network title

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
This is the last PR updating our dashboard, and we are also updating some titles to reduce their length. The following areas are affected with this PR:

1. Swap
2. Cachestat
3. Directory cache
4. Socket
5. Shared memory
6. IRQ

##### Test Plan

1. Clone this branch
2. Compile PR.
3. Start netdata, open your browser and access local dashboard. Force a reload to get the latest dashboard.
4. Access the links.

##### Additional Information
It is not necessary to test on different kernels.

<details> <summary>For users: How does this change affect me?</summary>
Describe the PR affects users: 
- Which area of Netdata is affected by the change? Dashboard
- Can they see the change or is it an under the hood? If they can see it, where? Yes, when they access eBPF charts.
- How is the user impacted by the change?  They will have a simpler way to navigate among charts.
- What are there any benefits of the change?  After this PR is merged, all charts delivered with `ebpf.plugin` will be connected.
</details>

### CODE REVIEW

File: collectors/ebpf.plugin/ebpf_shm.c [hunk 1]
```diff
@@ -988,7 +988,7 @@ static void ebpf_create_shm_charts(int update_every)
     ebpf_create_chart(
         NETDATA_EBPF_SYSTEM_GROUP,
         NETDATA_SHM_GLOBAL_CHART,
-        "Calls to shared memory system calls.",
+        "Calls to system calls.",
```
Comment (vlvkobal) [inline collectors/ebpf.plugin/ebpf_shm.c:991]:
```suggestion
        "Calls to system calls",
```

File: collectors/ebpf.plugin/ebpf_swap.c [hunk 2]
```diff
@@ -644,7 +644,7 @@ static void ebpf_swap_allocate_global_vectors(int apps)
 static void ebpf_create_swap_charts(int update_every)
 {
     ebpf_create_chart(NETDATA_EBPF_SYSTEM_GROUP, NETDATA_MEM_SWAP_CHART,
-                      "Calls to internal functions used to access swap.",
+                      "Calls to internal functions.",
```
Comment (vlvkobal) [inline collectors/ebpf.plugin/ebpf_swap.c:647]:
```suggestion
                      "Calls to internal functions",
```

File: collectors/ebpf.plugin/ebpf_mount.c [hunk 3]
```diff
@@ -386,7 +386,7 @@ static void mount_collector(ebpf_module_t *em)
 static void ebpf_create_mount_charts(int update_every)
 {
     ebpf_create_chart(NETDATA_EBPF_MOUNT_GLOBAL_FAMILY, NETDATA_EBPF_MOUNT_CALLS,
-                      "Calls to mount and umount syscalls.",
+                      "Calls to syscalls",
```
Comment (vlvkobal) [inline collectors/ebpf.plugin/ebpf_mount.c:389]: Do you really want to make titles look like units?

Comment (thiagoftsm) [reply collectors/ebpf.plugin/ebpf_mount.c]: We want to simplify them. I returned to previous version here.

File: collectors/ebpf.plugin/ebpf_shm.c [hunk 4]
```diff
@@ -988,7 +988,7 @@ static void ebpf_create_shm_charts(int update_every)
     ebpf_create_chart(
         NETDATA_EBPF_SYSTEM_GROUP,
         NETDATA_SHM_GLOBAL_CHART,
-        "Calls to shared memory system calls.",
+        "Calls to system calls",
```
Comment (vlvkobal) [inline collectors/ebpf.plugin/ebpf_shm.c:991]: This one is also simplified too much.

File: collectors/ebpf.plugin/ebpf_swap.c [hunk 5]
```diff
@@ -644,7 +644,7 @@ static void ebpf_swap_allocate_global_vectors(int apps)
 static void ebpf_create_swap_charts(int update_every)
 {
     ebpf_create_chart(NETDATA_EBPF_SYSTEM_GROUP, NETDATA_MEM_SWAP_CHART,
-                      "Calls to internal functions used to access swap.",
+                      "Calls to internal functions",
```
Comment (vlvkobal) [inline collectors/ebpf.plugin/ebpf_swap.c:647]: This one is also simplified too much.

Comment (thiagoftsm) [reply collectors/ebpf.plugin/ebpf_swap.c]: I updated both. Thanks!

**Review by vlvkobal — APPROVED** [review]

File: collectors/ebpf.plugin/ebpf_dcstat.c [hunk 6]
```diff
@@ -160,7 +160,7 @@ void ebpf_dcstat_create_apps_charts(struct ebpf_module *em, void *ptr)
 {
     struct target *root = ptr;
     ebpf_create_charts_on_apps(NETDATA_DC_HIT_CHART,
-                               "Percentage of files listed inside directory cache",
+                               "Percentage of files inside directory cache",
```
Comment (erdem2000) [inline collectors/ebpf.plugin/ebpf_dcstat.c:163]:
```suggestion
                               "Percentage of files inside the directory cache",
```
There are other updates without "the" article.  Obviously adding it is not crucial but just a suggestion.

**Review by ilyam8 — APPROVED** [review]

**Review by erdem2000 — APPROVED** [review]

**Pull request merged**